google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/JobStatistics.java

```diff
@@ -437,14 +448,24 @@ public Long getTotalBytesProcessed() {
       return queryPlan;
     }
 
+
+    /**
+     * Returns the schema for the query result. Present only for successful dry run of
+     * non-legacy SQL queries.
```

Comment:
I think this will be populated for all queries, not just dry runs right?

Reply:
I copied this from the API docs - I’ll check and report back.

Reply:
@mikekap You're absolutely right. I didn't read the docs closely enough.

Reply:
yep - just checked, the schema is `null` for a real query. In case of a real result `TableResult` has a schema.